src/main/java/uk/gov/hmcts/reform/sscs/helper/HearingsMapping.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.sscs.helper;
+
+import uk.gov.hmcts.reform.sscs.model.HearingWrapper;
+import uk.gov.hmcts.reform.sscs.model.single.hearing.HearingDeleteRequestPayload;
+
+public class HearingsMapping {
+
+    private static String CANCEL_REASON_TEMP = "AWAITING_LISTING";
+
+    private HearingsMapping() {
+
+    }
+
+    public static HearingDeleteRequestPayload buildDeleteHearingPayload(HearingWrapper wrapper) {
+        return HearingDeleteRequestPayload.builder()
+            .cancellationReasonCode(CANCEL_REASON_TEMP)
```

Comment:
This needs to use the LOV from [SSCS-10273 #1138](https://github.com/hmcts/sscs-common/pull/1138/files#diff-f06463c469e06696b9f17ec6a6ab9fdeeee4dd0d00b1dbe9ccd630d09d12dab2R9-R21).
Will this be sent over in the queue?

Reply:
Is there any harm to leaving the code there as I can leave a comment to and it would be as simple as just replacing CANCEL_REASON_TEMP which was advised to be added by Sean.